### ISSUE TITLE
Add Safe mode

### ISSUE THREAD

**Issue by YomostXme** [sanglantes/mean-chess#1]
Suggestion: You can add safe mode that display best move on screen (ex: Best move h4) or auto move because lichess can detect draw arrow and can flag you really fast

**Comment by sanglantes** [comment]
Yes, I was wondering whether Lichess anti-cheat measures were detecting arrows, but I doubt it. Nevertheless, I like the idea.
